ui/App.svelte

```diff
@@ -109,10 +109,6 @@
 
   <!-- TODO(julien): Dress up loading screen -->
   <div slot="loading" class="error">
-    {#if $location === path.lock()}
-      <EmptyState headerText="Unlocking the app..." emoji="🚪" text="" />
-    {:else}
-      <EmptyState headerText="Loading..." emoji="🕵️" text="" />
-    {/if}
+    <EmptyState headerText="Loading..." emoji="🕵️" text="" />
```

Comment:
I wonder if there's a way to get rid of this intermediate loading screen completely and only show the unlock screen while unlocking the app?

![loading](https://user-images.githubusercontent.com/158411/98254068-1c87ce00-1f7c-11eb-8d17-6daa447a32ae.gif)

Reply:
The intermediate screen comes from the session store being in state loading. Before, the remote slot for that state was just empty ie a blank screen. So should the session request take longer, there is some description of what is happening. @juliendonck do you think the screen is unnecessary now that the requests should be quick?

Reply:
Hm, I guess there is no way for the app to know what the intermediate state is before it's unlocked. I guess leaving the loading state then is probably the better option, in case the unlocking takes longer than normal, the user at least will know what's going on.